platform/retarget.cpp

```diff
@@ -266,6 +297,17 @@ extern "C" int PREFIX(_write)(FILEHANDLE fh, const unsigned char *buffer, unsign
 #endif
 #endif
         n = length;
+#if defined MBED_CFG_DEBUG_OPTIONS_COVERAGE
+    } else if (coverage_report && fh == gcov_fd){
+        for (unsigned int i = 0; i < length; i++) {
+            if (0 == buffer[i]){
+                printf(".");
+            } else {
+                printf("%02x", buffer[i]);
+            }
+        }
+        n = length;
+#endif
```

Comment:
Nit: Same deal in the above block about spaces, there should be spaces after the `if ()`.

Ex. `if (blah){` should be `if (blah) {`.